packages/ui/src/Markdown.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import ReactMarkdown, {ReactMarkdownProps} from 'react-markdown';
+import {Prism as SyntaxHighlighter} from 'react-syntax-highlighter';
+import {makeStyles} from '@material-ui/core/styles';
+
+const useStyles = makeStyles(
+    (theme) => ({
+        root: {
+            '& h1': {
+                ...theme.typography.h1,
+                fontSize: '1.8em',
+                borderBottom: 0,
+            },
+            '& a, & a code': {
+                color: theme.palette.primary.main,
```

Comment:
Does `theme.palette.primary.main` contain a blue color code? I don't understand why it's named like this, but if it works. :smile:

Reply:
Yes this is customizable via themes see https://material-ui.com/customization/theming

by default it is blue.